common-template/src/main/java/com/redhat/cloud/notifications/qute/templates/mapping/Console.java

```diff
@@ -0,0 +1,44 @@
+package com.redhat.cloud.notifications.qute.templates.mapping;
+
+import com.redhat.cloud.notifications.qute.templates.TemplateDefinition;
+import java.util.Map;
+
+import static com.redhat.cloud.notifications.qute.templates.IntegrationType.DRAWER;
+import static java.util.Map.entry;
+
+
+public class Console {
+    static final String BUNDLE_NAME = "console";
+
+    static final String INTEGRATIONS_FOLDER_NAME = "Integrations";
+
+    static final String RBAC_APP_NAME = "rbac";
+    static final String RBAC_FOLDER_NAME = "Rbac";
```

Comment:
General and not mandatory comment: I would move the `/` at the end of all folder names:
```suggestion
    static final String RBAC_FOLDER_NAME = "Rbac/";
```